source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,321 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to use **builders** to specify
+**sort criteria** for your queries.
+
+Sort criteria are the rules MongoDB uses to sort your data. Some examples of sort criteria are:
+
+* Smallest number to largest number
+* Earliest to latest time of day
+* Alphabetical order by first name 
+
+Builders are classes provided by the MongoDB Java driver that allow you to use
+the builder pattern to interact with your MongoDB instance.
+The builder pattern is an Object Oriented design technique for creating complex
+immutable objects separately from their representation. The builders provided
+by the MongoDB Java driver can improve the readability of your code.
+
+You should read this guide if you would like to use the builder pattern
+to specify sort criteria for your queries. If you are new to sorting in the
+MongoDB Java driver, consider reading our
+:doc:`CRUD Sort Guide </fundamentals/crud/read-operations/sort/>`
+for a general introduction to sorting with the MongoDB Java driver.
+
+The Sorts Class
+---------------
+
+The :java-docs:`Sorts <apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+class is the sort criteria builder provided by the MongoDB Java Driver.
+``Sorts`` provides static factory methods for all
+sort criteria operators supported by MongoDB. Each method returns
+an instance of the
+:java-docs:`Bson <apidocs/bson/org/bson/conversions/Bson.html?>`
```

Comment:
Let's not use ? to end URLs that don't contain any query params

Reply:
Thanks for catching this. Fix: https://github.com/mongodb/docs-java/pull/59/commits/baff37938847cae8e10f30c324e3755999aa930d

Going forward, I will try and make sure that my URLs are as clean as possible (as opposed to just functional).